mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/Bounty.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * DataLoader
+ * Copyright © 2021 SolarMC Developers
+ *
+ * DataLoader is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * DataLoader is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with DataLoader. If not, see <https://www.gnu.org/licenses/>
+ * and navigate to version 3 of the GNU Affero General Public License.
+ */
+
+package gg.solarmc.loader.kitpvp;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+/**
+ * The total bounty on a certain user
+ *
+ */
+public class Bounty {
+
+    private final String target;
```

Comment:
Why string and not a more descriptive object?

Reply:
We only need to fetch from the database what we need to display a bounty list. I can't imagine other details like the player UUID or solar ID being shown in the bounty list.

Reply:
We really only need the bounty list to display it in some user interface. I don't expect anything other than usernames to be used.